### ISSUE TITLE
[4.0] Login field error text colour fails a11y

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#23790]
### What needs to be fixed

The colour of the "Field value cannot be empty" text against the blue background being used fails WCAG AA

![screenshot_2019-02-08 joomla 4 - administrator login](https://user-images.githubusercontent.com/2019801/52468019-4f224e00-2b7f-11e9-8313-08f57a2b6683.png)

The colour should be `#d4d4d4` or **lighter**

**Comment by hardik-codes** [comment]
I'm working on this @C-Lodder

**Comment by hardik-codes** [comment]
are the changes to be made ```administrator/templates/atum/css/template.min.css``` in this file ?

**Comment by C-Lodder** [comment]
No, the SCSS file

**Comment by hardik-codes** [comment]
Which one could you please help me with that?

**Comment by C-Lodder** [comment]
@hardik-codes - https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/scss/blocks/_login.scss#L61

**Comment by hardik-codes** [comment]
how did you get to know that in which file are the changes to be made?

**Comment by C-Lodder** [comment]
Cause I helped build the v1 of the template and the file structure has pretty much stayed the same.

You can also inspect the element and do a mass `search and find` with your editor in the `administrator/templates/atum/scss` directory

**Comment by C-Lodder** [comment]
Pending PR: https://github.com/joomla/joomla-cms/pull/23831
